### DISCUSSION TITLE
Creating Admin User

### DISCUSSION THREAD

**Discussion by NickAtkinson** [socioboard/Socioboard-5.0#339] [Q&A]
I believe I have Socioboard set up on my server, with some additional configuration required for setting up the API credentials etc for most of the social media platforms. (Plus what I assume is going to be a lot of work getting it to use https instead of http.)

I have several questions, but at this point, my most pressing is: How do I set up an admin user? There's a flag in the mySQL DB for is_admin_user that is set to 0 by default, but it doesn't seem to do anything if I set it to 1.

My next most pressing question is: Is Socioboard 5.0 functional?

**Comment by sureshbabu1995** [comment]
Hi @NickAtkinson 

Yes there's a flag to set for admin `is_admin_user` in `user_details` table 

and second one -- Yes it's functional and we've our official application as well,  Application **landing page** is [SocioBoard](https://socioboard.com/) and our Application **login page is** - [SocioBoard Login](http://appv5.socioboard.com/)

**Reply by NickAtkinson** [reply]
I have is_admin_user in user_details set to 1, and it doesn't seem to grant the user admin permissions as far as I can tell. Is there another setting somewhere that will set Socioboard to use that flag instead of, say, using aMember? Or rather, are there other settings that I'd need to set to get it to work properly? Or is there another value that I need to set it to?

I'm missing something, and I'm not sure what it is at this point.
